### PR TITLE
#159961946 Integrate react-router

### PR DESCRIPTION
**Pull request by denniswanjiru**
#### What does this PR do?
Integrates react router to the app

#### Description of Task to be completed?
- Add react-router-dom to the app
- Create dummy components to test routing
- Write tests for the dummy components
- Add sass from the mockups
- Add PULL_REQUEST_TEMPLATE
- Delete the unnecessary folders

#### How should this be manually tested?
```
git clone git@github.com:andela/ah-jarvis-frontend.git  -b ch-react-router-159961946

cd ah-jarvis-frontend

yarn 

yarn sass:build

yarn start

```
navigate to localhost:3000/signin

#### Screenshots (if appropriate)
What you should expect
screen shot 2018-09-03 at 14 40 20

#### What are the relevant pivotal tracker stories?
[# 159961946](https://www.pivotaltracker.com/story/show/159961946)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+  <!--
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <!--
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css">
```
Comment (yeenab) [inline public/index.html]: Can we add materialize as an npm package rather than a CDN

File: package.json [hunk 2]
```diff
@@ -33,5 +34,13 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.11.1"
+  },
```
Comment (yeenab) [inline package.json:38]: In the future, if you have something such as *configs* that is not related to the current PR. Create a new story for it and a different PR.

File: src/styles/scss/index.scss [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+@import "components/buttons";
+@import "components/cards";
+@import "components/comments";
+@import "components/icons";
+@import "components/nav";
+@import "layout/navbar";
+@import "pages/auth";
+@import "pages/articles";
+@import "utils/helpers";
+@import "pages/profile";
```
Comment (yeenab) [inline src/styles/scss/index.scss]: We have way too many css helpers, that we don't for now. Again, the task here is setting up `react-router`. I don't see how these files are helping us for now. If you would like to have basic styling written, create a chore.

**Review by yeenab — CHANGES_REQUESTED** [review]
Way too much code for just `Setup Routing`

File: public/index.html (hunk 1 above)
Comment (denniswanjiru) [reply public/index.html]: Yeah, I will do that

File: src/styles/scss/index.scss (hunk 3 above)
Comment (denniswanjiru) [reply src/styles/scss/index.scss]: okay I will remove them. We had them wired during the mockup and our workflow integrates perfectly with it

File: package.json (hunk 2 above)
Comment (denniswanjiru) [reply package.json:38]: Okay, I have noted that. Thanks for the insights.

File: src/styles/css/index.css [hunk 4]
```diff
@@ -1,2 +1,870 @@
-h1 {
-  color: #0fe05f; }
+@import url("https://fonts.googleapis.com/css?family=Open+Sans");
+html,
+body,
```
Comment (yeenab) [inline src/styles/css/index.css]: @DennisWanjiru I still think we don't need this file at the moment. Let's avoid going ahead of our selves and write code to be used for the moment.

**Review by yeenab — CHANGES_REQUESTED** [review]

File: src/styles/css/index.css (hunk 4 above)
Comment (denniswanjiru) [reply src/styles/css/index.css]: Yeah @j33n. I forgot to recompile the sass. Yeah, am removing it right away. Thanks for the highlight.

**Review by yeenab — APPROVED** [review]

**Pull request merged**